app/src/main/java/com/codesoom/assignment/MyHttpHandler.java

```diff
@@ -0,0 +1,151 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.models.Task;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.List;
+
+import static com.codesoom.assignment.HttpStatusCode.*;
+
+public class MyHttpHandler implements HttpHandler {
+
+    JSONConverter jsonConverter = new JSONConverter();
+    TaskRepository taskRepository = new TaskRepository();
+    private Long idInPath;
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+
+        HttpRequest httpRequest = new HttpRequest(exchange);
+        String content = "Hello, world!";
+
+
+        if (!httpRequest.hasMethod().isEmpty()) {
+            switch (httpRequest.hasMethod()) {
+                case "GET":
+                    // /tasks만 입력했을 경우 (아이디로 검색하지 않고 전체 목록 얻을 때)
+                    if (httpRequest.hasPath().equals("/tasks")) {
+                        GETAllTaskList();
+                        content = JSONConverter.tasksToJSON(TaskRepository.getTaskStore());
+                        response(HTTP_OK, content, exchange);
+                        break;
+                    }
+                    // tasks/{id}를 입력했을 경우 (입력한 아이디에 해당하는 할일만 보여줌)
+                    Long id = getIdFromPath(httpRequest);
+                    findOne(id);
```

Comment:
이 코드는 아무런 일을 하고 있지 않은 것 같아요. 어디에 쓰이는건가요?

Reply:
```suggestion
                    /* /tasks/{id}를 입력했을 경우 (입력한 아이디에 해당하는 할 일만 가져오는 경우) */
                    Long id = httpRequest.getId();
                    content = jsonConverter.taskToJson(findTaskById(id));
                    response(HTTP_OK, content, exchange);
                    break;
```

findOne(id); 코드는 잘못 적은것 같습니다! 위와 같이 수정해 보았습니다.